lib/DiscoveryStreamFeed.jsm

```diff
@@ -141,10 +143,17 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
         throw new Error(`Not one of allowed prefixes (${allowed})`);
       }
 
-      const response = await fetch(endpoint, {credentials: "omit"});
+      const controller = new AbortController();
+      const {signal} = controller;
+      const fetchPromise = fetch(endpoint, {credentials: "omit", signal});
+      // istanbul ignore next
```

Comment:
Without istanbul ignore code coverage complains test coverage for this line https://github.com/mozilla/activity-stream/pull/4971/files#diff-8b1b03c4d5dec5c8e4f7de50e05c0c4fR185

Do you know why istanbul ignore next help fix that, will be good to add as comment here

Reply:
It seems code coverage is wrongly pointing to isExpired, added test covers that edge case